### PR TITLE
zh: update outdated link in RBAC tasks

### PR DESCRIPTION
**Pull request by yangminzhu**
Fixes https://github.com/istio/istio.io/issues/4417

### CODE REVIEW

File: content_zh/docs/tasks/security/authz-tcp/index.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ keywords: [security,access-control,rbac,tcp,authorization]
     * 创建新的 `ServiceAccount` 并在**启用自动注入**的网格中部署新版本的服务：
 
         {{< text bash >}}
-        $ kubectl apply -f samples/bookinfo/platform/kube/rbac/ratings-v2-add-serviceaccount.yaml
+        $ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-ratings-v2.yaml@
         {{< /text >}}
 
     * 创建新的 `ServiceAccount` 并在**没有启用自动注入**的网格中部署新版本的服务：
```
Comment (sdake) [inline content_zh/docs/tasks/security/authz-tcp/index.md]: this section seems duplicated?

Comment (yangminzhu) [reply content_zh/docs/tasks/security/authz-tcp/index.md]: This one is for manual sidecar injection (`istioctl kube-inject ...`), the above one is for auto sidecar injection.

**Pull request merged**